doc/alertmanager-mixin/Makefile

```diff
@@ -0,0 +1,22 @@
+JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
+ALERTMANAGER_ALERTS := alertmanager_alerts.yaml
+
+all: fmt $(ALERTMANAGER_ALERTS) lint
+
+fmt:
+	find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
+		xargs -n 1 -- $(JSONNET_FMT) -i
+
+$(ALERTMANAGER_ALERTS): mixin.libsonnet alerts.libsonnet
+	jsonnet -S alerts.jsonnet > $@
+
+lint: $(ALERTMANAGER_ALERTS)
+	find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
+		while read f; do \
+			$(JSONNET_FMT) "$$f" | diff -u "$$f" -; \
+		done
+
+	promtool check rules $(ALERTMANAGER_ALERTS)
```

Comment:
Do you mind if we use mixtool here?  Happy to make the change myself.

Reply:
Done.